src/components/playlist/playlistPanel.tsx

```diff
@@ -0,0 +1,78 @@
+import {Icon, Menu} from 'antd';
+import {push} from 'connected-react-router';
+import {map} from 'lodash';
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {addPlaylist} from '../../actionCreators';
+import {getActivePlaylist, getAllPlaylist} from '../../reducers/playlist';
+import {getActiveSong, getAllSongs} from '../../reducers/songs';
+import Browser from '../browser';
+import AddPlaylistModal from './addPlaylistModal';
+
+
+class PlaylistView extends React.Component<{
```

Comment:
I think this might be a great chance to try `react-hooks`. Read about it and try it out.